src/library.js

```diff
@@ -1409,7 +1409,7 @@ LibraryManager.library = {
   // ==========================================================================
 
   _emscripten_throw_longjmp__sig: 'v',
-  _emscripten_throw_longjmp: function() { throw 'longjmp'; },
+  _emscripten_throw_longjmp: function() { throw Infinity; },
```

Comment:
I wonder if the existence of this whole function should be gated behind

```
#if SUPPORT_LONGJMP == 'emscripten'
```
?

Though the function

```c
void emscripten_longjmp(uintptr_t env, int val) {
  setThrew(env, val);
  _emscripten_throw_longjmp();
}
```
in emscripten_setjmp.c is not gated behind `#ifdef __USING_WASM_SJLJ__`, which seems to be at odds against the choice with line 709 in preamble.js (`#if SUPPORT_LONGJMP == 'emscripten'`)?

Reply:
Those both seem like reasonable suggestions.  Neither should save any code size benefit I guess because neither should ever be referenced in that case.. but its still good for correctness to prevent accidental references.